settings.gradle

```diff
@@ -11,6 +11,7 @@ include ':brightcove-exoplayer:AdRulesIMASampleApp'
 include ':brightcove-exoplayer:AdRulesIMAWidevineModularSampleApp'
 include ':brightcove-exoplayer:AppCompatActivitySampleApp'
 include ':brightcove-exoplayer:AppCompatFragmentSampleApp'
+include ':brightcove-exoplayer:BasicSsaiSampleApp'
 include ':brightcove-exoplayer:BasicOnceUxSampleApp'
```

Comment:
Since we've deprecated the OnceUx plugin, do we want to also remove this line from settings.gradle? Perhaps remove the OnceUx sample app altogether?